### ISSUE TITLE
Obfuscated classes cannot be loaded [ASM crash]

### ISSUE THREAD

**Issue by skidunion** [Col-E/Recaf#235]
**Describe the bug**
From https://github.com/java-deobfuscator/deobfuscator/issues/600
Another kind of an ASM crasher. Classes appear in the class view, but cannot be opened.

**To Reproduce**

1. Open the attached sample in Recaf
2. Open a class in the class tree
3. The class would appear as a resource

**Exception**
The exception happens while parsing the **NestedMembers** attribute, specifically while trying to parse the class that is referenced.
```
14:42:31.376 [ForkJoinPool-1-worker-9] INFO : Unknown protection on class file
14:42:31.377 [ForkJoinPool-1-worker-9] WARN : Invalid class "ps.class" - Cannot be parsed with ASM reader
Adding as a file instead.
14:42:31.377 [ForkJoinPool-1-worker-9] INFO : Unknown protection on class file
14:42:31.377 [ForkJoinPool-1-worker-9] WARN : Invalid class "Rk.class" - Cannot be parsed with ASM reader
Adding as a file instead.
14:42:31.377 [ForkJoinPool-1-worker-9] INFO : Unknown protection on class file
14:42:31.377 [ForkJoinPool-1-worker-9] WARN : Invalid class "SL.class" - Cannot be parsed with ASM reader
```

**Comment by Col-E** [comment]
> The exception happens while parsing the NestedMembers attribute

Well, the major version of the classes seems to be `52` == Java 8... Nests were added in Java 11 so that's clearly illegal.
Plus, there's more than that wrong with the class, anyways, will add in some patchwork.

**Comment by Col-E** [comment]
Added help wanted because my time is quite limited, but I'm working on a bytecode parser library to add multiple features to Recaf... In this case, it can be used to filter out illegal code attributes and other intentional malformed objects from classes before Recaf passes them the the ASM parser.

https://github.com/Col-E/CAFED00D

Most of the work necessary is just reading the official class file specification and implementing it. There are test cases to validate progress.

**Comment by Col-E** [comment]
Most of this got addressed in 2.18 which included the stripping feature of cafedood.

**Comment by x4e** [comment]
:/
